docs/intentReaderHandlers/LifecycleRecordHandler.md

```diff
@@ -0,0 +1,205 @@
+# LifecycleRecordHandler
+
+## Overview
```

Comment:
Change line from _**FCA starts listening for lifecycle events and will stop listening once it gets the intent with the task "StopLifecycleRecording", and the event list will be saved.**_ to _**FCA will start listening to lifecycle events and will stop listening once it gets the intent with the task "StopLifecycleRecording". The list of events received in the time frame is saved in a list and returned**_